roles/ovirt-cluster-upgrade/library/ovirt_clusters.py

```diff
@@ -0,0 +1,567 @@
+#!/usr/bin/python
```

Comment:
It will be only 2.4, because we added here a new feature and new features can't go to Z-stream.